src/app_model/backends/qt/_qkeymap.py

```diff
@@ -250,30 +264,33 @@ def __init__(self, kb: KeyBinding) -> None:
 
 def qmods2modelmods(modifiers: Qt.KeyboardModifier) -> KeyMod:
     """Return KeyMod from Qt.KeyboardModifier."""
-    swapped = None
     mod = KeyMod.NONE
-    if modifiers & Qt.KeyboardModifier.ShiftModifier:
-        mod |= KeyMod.Shift
-    if modifiers & Qt.KeyboardModifier.ControlModifier:
-        swapped = MAC and mac_ctrl_meta_swapped()
-        if swapped:
-            mod |= KeyMod.WinCtrl
-        else:
-            mod |= KeyMod.CtrlCmd
-    if modifiers & Qt.KeyboardModifier.AltModifier:
-        mod |= KeyMod.Alt
-    if modifiers & Qt.KeyboardModifier.MetaModifier:
-        if swapped is None:
-            swapped = MAC and mac_ctrl_meta_swapped()
-        if swapped:
-            mod |= KeyMod.CtrlCmd
-        else:
-            mod |= KeyMod.WinCtrl
+    lookup = (
+        MAC_KEYMOD_FROM_QT if MAC and not mac_ctrl_meta_swapped() else KEYMOD_FROM_QT
+    )
```

Comment:
I do not like this if. It could be resolved on import time. Execution of this if every function call looks like a bad pattern.

Reply:
I'm not sure that's true?  `Qt.AA_MacDontSwapCtrlAndMeta` can change dynamically.

```python
In [3]: from qtpy.QtCore import Qt, QCoreApplication

In [6]: app = QCoreApplication([])

In [7]: app.testAttribute(Qt.AA_MacDontSwapCtrlAndMeta)
Out[7]: False

In [8]: app.setAttribute(Qt.AA_MacDontSwapCtrlAndMeta)

In [9]: app.testAttribute(Qt.AA_MacDontSwapCtrlAndMeta)
Out[9]: True
```

i'm ok leaving that possible performance improvement for later